example_4_word_cloud/main.py

```diff
@@ -0,0 +1,34 @@
+import os
+from collections import Counter
+
+from wordcloud import WordCloud
+import matplotlib.pyplot as plt
+from speakleash import Speakleash
+import spacy
+
+base_dir = os.path.join(os.path.dirname(__file__))
```

Comment:
should be ' __file__' instead of __file__

Reply:
Did a test (Windows 10, python 3.10.7, VS Code):
```python
base_dir = os.path.join(os.path.dirname('__file__'))
print(f"With \' characters -> {base_dir=}")

base_dir = os.path.join(os.path.dirname(__file__))
print(f"Without \' characters -> {base_dir=}")
```

Output:
```
With ' characters -> base_dir=''
Without ' characters -> base_dir='e:\\GitHub\\SpeakLeash_PL_DATA\\speakleash-examples\\example_5_word_cloud'
```